gradebook/src/GradeBook/book.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+
+/*
+The class Book packages the blueprint book provides ways to:
+1 store the grades. 
+2 add to the given gradebook obj
+3 welcomes the user from args
+4 Assign name to the gradebook
+ */
+
+namespace GradeBook
+{
+    public class book
+    {
+        List<double> grades;
+        string name;
+        double highGrade = double.MinValue;
+        double lowGrade = double.MaxValue;
+        double avg = 0.0;
+
+        //Constructor
+        public book(string name, string[] args) 
+        {
+            grades = new List<double>() {1,22,56,33.2,45,30,32};
+            this.name = name;
+            welcomeUser(args);
+        }
+
+        public void calculateStats()
+        {
+            foreach(double gd in this.grades)
```

Comment:
This is more efficient in this case but if you haven't run into LINQ yet it'd be worth looking at for the future.

Reply:
Sure I will keep that in mind.